pkg/fakedata/generator.go

```diff
@@ -203,6 +203,12 @@ var file = func(column Column) string {
 	return list[rand.Intn(len(list))]
 }
 
+var corporaGenerator = func(source []string) func(Column) string {
+	return func(column Column) string {
+		return source[rand.Intn(len(source))]
+	}
+}
+
```

Comment:
While working on something else I remembered I didn't comment here. 

I think you can use `withList`, it has the exact same behavior (I'm removing `withEnum` for the very same reason)